src/openrct2/ride/Track.cpp

```diff
@@ -1992,35 +2223,37 @@ void game_command_set_maze_track(sint32 *eax, sint32 *ebx, sint32 *ecx, sint32 *
  *
  *  rct2: 0x006C5AE9
  */
-void game_command_set_brakes_speed(sint32 *eax, sint32 *ebx, sint32 *ecx, sint32 *edx, sint32 *esi, sint32 *edi, sint32 *ebp)
+void game_command_set_brakes_speed(sint32 * eax, sint32 * ebx, sint32 * ecx, sint32 * edx, sint32 * esi, sint32 * edi, sint32 * ebp)
 {
-    rct_map_element *mapElement;
-    sint32 x, y, z, trackType, brakesSpeed;
+    rct_map_element * mapElement;
+    sint32          x, y, z, trackType, brakesSpeed;
```

Comment:
alignment should match variable names, not type specifiers, i.e.
```
rct_map_element * mapElement;
sint32            x;
```
not 
```
rct_map_element * mapElement;
sint32          x;
```

Reply:
I know, I already fixed a few. It's the autoformatter,